### PR TITLE
Add types for https://github.com/jaredhanson/passport-anonymous

### PR DESCRIPTION
**Pull request by 0x6368656174**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: passport-anonymous/passport-anonymous-tests.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+import login = require("passport-anonymous");
```
Comment (mhegazy) [inline passport-anonymous/passport-anonymous-tests.ts]: The test should have a more content, something like:

```ts
import * as passport from "passport";
import  { Strategy } from "passport-anonymous";

passport.use(new Strategy());
```

File: passport-anonymous/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Type definitions for passport-anonymous 1.0
+// Project: https://github.com/jaredhanson/passport-anonymous
+// Definitions by: Pavel Puchkov <https://github.com/0x6368656174>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import passport = require("passport");
+
+declare class Strategy implements passport.Strategy {
```
Comment (mhegazy) [inline passport-anonymous/index.d.ts]: this does not look accurate.. see https://github.com/jaredhanson/passport-anonymous/blob/master/lib/index.js#L15


the module should have `export = Strategy` at the bottom, and `Strategy` should have a static member `static Strategy: typeof Strategy`

**Review by mhegazy — CHANGES_REQUESTED** [review]

File: passport-anonymous/passport-anonymous-tests.ts (hunk 1 above)
Comment (0x6368656174) [inline passport-anonymous/passport-anonymous-tests.ts]: fix

File: passport-anonymous/index.d.ts (hunk 2 above)
Comment (0x6368656174) [inline passport-anonymous/index.d.ts]: I did it like other passport packages:
- passport-beam: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-beam/index.d.ts#L12-L16
- passport-facebook: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-facebook/index.d.ts#L32-L37
- passport-http: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-http/index.d.ts#L45-L51
- passport-local: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-local/index.d.ts#L37-L44
- passport-google-oauth: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/passport-google-oauth/index.d.ts#L65-L70
- others....

**Pull request merged**